py/mkrules.mk

```diff
@@ -201,7 +201,7 @@ endif
 
 all: $(BUILD)/$(PROG)
 
-$(BUILD)/$(PROG): $(OBJ)
+$(BUILD)/$(PROG): $(OBJ) $(EXTRA_OBJ)
```

Comment:
Instead of doing this here it can be done in `ports/windows/Makefile`:
```
$(BUILD)/$(PROG): $(BUILD)/micropython.res
```
That should just append the .res to the list of dependencies for the executable, and it'll be included in `$@`.